src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
As per google codestyle guide, all constants should use `CONSTANT_CASE` - all letters uppercase, separated with a single underscore